apps/daffio/src/app/core/nav/docs/docs.component.html

```diff
@@ -11,10 +12,16 @@
       }
     </ng-container>
   }
-  <daffio-docs-search-button></daffio-docs-search-button>
-  <daff-theme-switch-button></daff-theme-switch-button>
-  <daffio-docs-search-button search-mobile [icon]="true"></daffio-docs-search-button>
-  <daff-theme-switch-button theme-toggle-mobile></daff-theme-switch-button>
+	@if (isBigTablet) {
+		<daffio-docs-search-button></daffio-docs-search-button>
+	} @else {
+		<daffio-docs-search-button search-mobile [icon]="true"></daffio-docs-search-button>
+	}
+	@if (isBigTablet) {
+		<daff-theme-switch-button></daff-theme-switch-button>
+	} @else {
+		<daff-theme-switch-button theme-toggle-mobile></daff-theme-switch-button>
+	}
```

Comment:
@xelaint this fixes the double modal issue. Its not a good idea to have components in the DOM that aren't being used, its quite wasteful and will cause perf issues.